CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## v0.4.0
+
+* Relax gem constraint for Active Support dependency #48
```

Comment:
I support the idea of the changelog in general, but while we're just going into closed beta it feels like a bunch of just...extra work for no real value? I think perhaps I'd be more keen to not have one for now, and when we got out of closed beta we could do a `1.0.0` release of this gem and start changelogging from there?

Reply:
sure thing 👍🏼